Dockerfiles/ci/command.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+OPERATOR_DIR="${OPERATOR_DIR:-/project/operator_dir/}"
+OPERATOR_WORK_DIR="${OPERATOR_WORK_DIR:-/project/test_operator_work_dir/}"
+WORK_DIR="${WORK_DIR:-/project/output/}"
+
+test_prepare_operator_metadata () {
+    ansible-playbook -vvv -i localhost, --connection local operator-test-playbooks/prepare-operator-metadata.yml -e "operator_dir=$OPERATOR_DIR" -e "operator_work_dir=$OPERATOR_WORK_DIR" -e"work_dir=$WORK_DIR"
+    cp /project/operator-test-playbooks/parse_operator_metadata_results.json $WORK_DIR
+    test -f $WORK_DIR/parsed_operator_data.yml
+    test -f $WORK_DIR/parse_operator_metadata_results.json
+
+}
+
+
+test_for_report_failed_empty_alm_examples() {
+    echo "test_for_report_failed_empty_alm_examples: This test checks for failure when empty alm examples are found"
+    grep -inr "Failed due to CSV.metadata.annotations\['alm-examples'\] is set to empty" $WORK_DIR
+    check=$?
+    echo "Exit code for grep $check"
+    if [ $check -eq 0 ]; then
+	echo "test_for_alm_examples Test passed"
+        exit 0
+    else
+	echo "test_for_alm_examples Test failed"
+        exit 1
+    fi	
+}
+
+
+TEST_NAME_ENV="${TEST_NAME:-NOTEST}"
+if [[ "$TEST_NAME_ENV" = 'test_for_report_failed_empty_alm_examples' ]]; then
```

Comment:
just a question. Should we run all tests when TEST_NAME_ENV is not specified?

IMHO it is bit unclear that `test_prepare_operator_metadata` was executed for
`-e TEST_NAME=test_for_report_failed_empty_alm_examples`

Reply:
@lslebodn Since we do not have any other test i have mentioned only 
```
test_for_report_failed_empty_alm_examples
```
I can default it to all tests if not mentioned.

Reply:
I have changed the default to run all the tests 
@lslebodn

Reply:
Thank you for changing default for missing env.

But Do we really need to run `test_prepare_operator_metadata` for `-e TEST_NAME=test_for_report_failed_empty_alm_examples`

I would expect just single test would be executed if I specify some name in env variable `TEST_NAME_ENV`

Reply:
At least that is how I understand the use-case from the name of variable,

If my understanding is not clear than you need to either update name of variables
or to better document the use-case

Reply:
I will document the flow. 
we need to run the test_prepare_operator_metadata since it would create all the output logs. 
I can make it a part of the test_for_report_failed_empty_alm_examples 
That can clear out the confusion. Further, 
I will re-name the  test_prepare_operator_metadata  to prepare_operator_metadata.

Reply:
Done. Added documentation. 
Please re-review. Thanks.

Reply:
It still does not make a sense to me.

Let say that we have 5 tests:
* test_for_report_failed_empty_alm_examples
* test_2
* test_3
* test_4
* test_5.

I would expect that all tests will be executed for missing env TEST_NAME_ENV
for  `e TEST_NAME_ENV=test_5` jsut the *test_5* will be executed.

With the current logic everything would be executed because `"test_5" != 'test_for_report_failed_empty_alm_examples'`

I am sorry but that does not make any sense to me.
I cannot see any benefit of this `if/else` block with he current implmentation

Reply:
If there are more tests there will be more elif statements inside commands.sh
By default, if there is no test specified, test_name_env is set to NOTEST and it will default to else block 
, and else block will run all tests. 
@lslebodn Did this clarify ? 
Shall i use a switch statement instead? 
how would you like this to be ?

Reply:
If the environment variable is not set 
line 31: TEST_NAME_ENV="${TEST_NAME:-NOTEST}"
 it would be initialized to 'NOTEST' value. 
 @lslebodn

Reply:
> If there are more tests there will be more elif statements inside commands.sh

Such approach does not scale.
I cannot imagine  very long `if/elif/else`block for 100+ tests